win32ss/user/ntuser/message.c

```diff
@@ -1727,6 +1728,9 @@ co_IntSendMessageNoWait(HWND hWnd,
    message parameters cannot include pointers. Otherwise, the operation will fail.
    The functions will return before the receiving thread has had a chance to
    process the message and the sender will free the memory before it is used.
+
+   Win: xxxSendMessageCallback(PWND, Msg, wParam, lParam,
+                               CompletionCallback, CompletionCallbackContext, uResult)
```

Comment:
Part of one's personal reversing notes; has no place in our codebase...

Reply:
Deleted.